stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/Call.kt

```diff
@@ -266,7 +266,7 @@ public class Call(
         response.onSuccess {
             state.updateFromResponse(it)
             if (ring) {
-                client.state.addRingingCall(this)
+                client.state.addRingingCall(this, RingingState.Outgoing())
```

Comment:
Is `Call.create` always called when initiating a direct call and is `ring` always set to `true`? I am relying on it to start the foreground service and pass `Outgoing` to it.

See `addRingingCall` [comment](https://github.com/GetStream/stream-video-android/pull/977/files#r1447059871) below.